src/Card/Card.jsx

```diff
@@ -91,7 +91,7 @@ Card.propTypes = {
   noPadding: PropTypes.bool,
   size: PropTypes.string,
   subTitle: PropTypes.string,
-  title: PropTypes.string,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
strings are technically nodes, so assuming prop-types is accurate, this can just be:

```suggestion
  title: PropTypes.node,
```

But probably worth double checking

Reply:
Done